content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -365,6 +365,43 @@ This applies only to significant, user-visible behaviors which impact the
 correctness of applications running on Kubernetes or that impact the
 administration of Kubernetes clusters, and which are being removed entirely.
 
+An exception to the above rule is **feature-gates**. Feature gates are key=value 
+pairs that allow for users to enable/disable experimental features.
+
+Feature gates are intended to cover the development life cycle of a feature - they 
+are not intended to be long-term APIs. As such, they are expected to be deprecated 
+and removed after a feature becomes GA or is dropped. 
+
+As a feature moves through the stages, the associated feature-gate will evolve 
+at the same time as announcing its deprecation. In general the life cycle is:
+
+  * The feature is Alpha - its feature gate is disabled by default and can be 
+  enabled by the user.
+  * The feature becomes Beta - its gate becomes enabled by default and can be 
+  disabled by the user.
+  * The feature becomes GA - its gate becomes a no-op and is deprecated.
+  * The feature becomes GA and a deprecation window is announced (see rule below) - its 
+  gate is removed and is no longer accepted.
+
+A feature might be EOL'ed at any point in the life cyle prior to GA.  In that case, 
+the associated feature-gate is deprecated. 
```

Comment:
I still like a sentence about accepting but ignoring.  I want no confusion...

What I proposed was:

Deprecated feature-gates for GA features are still accepted (i.e. do not cause errors) but do not have any effect.  Features that are intended to be user-disabled should include mechanisms for that, which are themselves covered by the feature-gate.

Reply:
To allow for a smooth transition from the sentence before I am changing from: 
"The gate can still be accepted but should not have any effect and should not cause any 
errors."
To:
"This is also true for deprecated feature-gates of GA features, hence the gate can still 
be accepted but should not have any effect and should not cause any errors. Features that 
are intended to be user-disabled should include mechanisms for that, which are themselves 
covered by the feature-gate."